packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItemGroup.jsx

```diff
@@ -90,7 +90,7 @@ class CollapsibleMenuViewItemGroup extends React.Component {
         });
         return (
           <li role="none">
-            <div {...customProps} role="group">
+            <div {...customProps} aria-label={ariaLabel} role="group">
```

Comment:
ariaLabel is not being added as first class prop here. Then why do we need explicit prop when Consumer can still pass in aria-label through custom props.